index.html

```diff
@@ -181,3 +182,13 @@
 <h2>Role of the editor</h2>
 
 <p>In-process additions will likely have spec text which is authored by a champion or a committee member other than the editor although in some case the editor may also be a champion with responsibility for specific features. The editor is responsible for the overall structure and coherence of the ECMAScript specification. It is also the role of the editor to provide guidance and feedback to spec text authors so that as an addition matures, the quality and completeness of its specification improves. It is also the role of the editor to integrate additions which have been accepted as “finished” (stage 4) into the a new revision of the specification.
+
+<h2>Browser implementations</h2>
+
+Browser implementations refer to web browsers such as Chromium, WebKit, Edge or Firefox. Implementations in web browsers often have additional constraints that other implementations don't have, such as a detailed understanding of web compatibility that comes from implementation so their experience is uniquely valuable. Specifically enfrancising browsers here, and differentiating them from transpilers, has the important benefit of making sure the committee doesn't add things to the specification that will never ship in the browser consensus.
```

Comment:
Chromium and WebKit seem like they're more in the same bucket as ChakraCore and SpiderMonkey; whereas Chrome and Safari would be in the same bucket as Edge or Firefox?

Also, what about non-transpiler non-browser native engines?